src/server/game/AuctionHouse/AuctionHouseMgr.h

```diff
@@ -85,6 +85,7 @@ struct TC_GAME_API AuctionEntry
     ObjectGuid::LowType bidder;
     uint32 deposit;                                         //deposit can be calculated only when creating auction
     uint32 etime;
+    std::unordered_set<uint32> bidderlist;
```

Comment:
if this is meant to be a guid, use ObjectGuid::LowType, never the underlying type (master branch portability issue, its a different type there)